agent/structs/structs.go

```diff
@@ -72,6 +72,43 @@ const (
 	SystemMetadataRequestType                   = 31
 )
 
+// requestTypeStrings is used for snapshot enhance
+// any new request types added must be placed here
+var requestTypeStrings = map[MessageType]string{
+	0:  "Register",
+	1:  "Deregister",
+	2:  "KVS",
+	3:  "Session",
+	4:  "ACL", // DEPRECATED (ACL-Legacy-Compat)
+	5:  "Tombstone",
+	6:  "CoordinateBatchUpdate",
+	7:  "PreparedQueryRequest",
+	8:  "TxnRequest",
+	9:  "AutopilotRequest",
+	10: "Area",
+	11: "ACLBootstrap",
+	12: "Intention",
+	13: "ConnectCA",
+	14: "ConnectCAProviderState",
+	15: "ConnectCAConfig", // FSM snapshots only.
+	16: "Index",           // FSM snapshots only.
+	17: "ACLTokenSet",
+	18: "ACLTokenDelete",
+	19: "ACLPolicySet",
+	20: "ACLPolicyDelete",
+	21: "ConnectCALeaf",
+	22: "ConfigEntry",
+	23: "ACLRoleSet",
+	24: "ACLRoleDelete",
+	25: "ACLBindingRuleSet",
+	26: "ACLBindingRuleDelete",
+	27: "ACLAuthMethodSet",
```

Comment:
Since these `Set` operations save an ACL resource, the type could be the name of the resource itself. It may make the table clearer.

That way the output would look like: `ACLToken 3 1.1KB` vs `ACLTokenSet 3 1.1KB`

```suggestion
	17: "ACLToken",
	18: "ACLTokenDelete",
	19: "ACLPolicy",
	20: "ACLPolicyDelete",
	21: "ConnectCALeaf",
	22: "ConfigEntry",
	23: "ACLRole",
	24: "ACLRoleDelete",
	25: "ACLBindingRule",
	26: "ACLBindingRuleDelete",
	27: "ACLAuthMethod",
```